public_html/wp-content/mu-plugins/blocks/source/block/organizers/block-controls.js

```diff
@@ -7,28 +7,31 @@ import classnames from 'classnames';
  * WordPress dependencies
```

Comment:
❤️

Reply:
I can't tell what you're heart-ing here. I'll just take it as a general sign of encouragement 😁

Reply:
Er, yeah, it makes more sense if you [view it in the context of the commit](https://github.com/WordPress/wordcamp.org/pull/117/commits/e9bcd2897aff6b1682a68496592134049d1d0e40), which is what I was looking at when I wrote it, but even then it's not entirely clear.

I was referring to the commit message (or, really, to the intent behind the commit, not the literal message, but that was fine too). It doesn't really apply to a specific line, but GitHub doesn't always let me comment on the commit as a whole (haven't noticed the pattern to when it does and doesn't, maybe it's only enabled for ones that are merged?)

Anyway, I should have quoted the commit message before my reply to make that clear :)